addons/website_sale/static/src/css/website_sale.css

```diff
@@ -663,3 +663,10 @@ a.no-decoration {
 .o_shadow_animation {
   box-shadow: 0 0 5px 10px rgba(240, 8, 0, 0.4) !important;
 }
+
+/* cart popover */
+.o_wsale_cart_popover_product_lines {
```

Comment:
@fja-odoo Clearly not a bug to fix in 11.0, it is just an improvement to prevent this to go out of the screen. And since it requires XML update, it is definitely not worth a stable fix.

Reply:
12.5